pages/views/roles/data.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Data View
+permalink: views/roles/data
+---
+
+# Data View
+
+{% include carbonStandard.html variant=site.data.overlays.data hideOutOfScope=true %}
+
+Every bit of data, at rest or in transit, incurs a carbon cost.
+If a user minimises the amount of data they store, then, they can significantly reduce carbon emissions due to storage. Doing so is also likely to reduce monetary costs: hosted services generally charge by data use, while local storage will use less energy to read and write the data, and can use fewer or smaller storage options, reducing embedded carbon.
+
+## Upstream Emissions
+
+## Operational Emissions
```

Comment:
Double H2 headings - can we reduce that to a single heading - "Upstream and Operational Emissions"?